cbmc/proofs/IotMqtt_PublishSync/Makefile

```diff
@@ -0,0 +1,60 @@
+ENTRY=IotMqtt_PublishSync
+
+# We abstract all the log and concurrency related functions in this proof
+# and assume their implementation is correct
+ABSTRACTIONS += --remove-function-body IotLog_Generic
+
+# We abstract this function because manual inspection suggest it is unreachable
```

Comment:
manual inspection demonstrates that it is unreachable.

Reply:
Done!